macchanger: add page (#4087)
tldr-pages/tldr: pages/linux/macchanger.md

# macchanger

> > Command line utility for manipulating network interface MAC addresses.

Reviewer: ```suggestion
> Command-line utility for manipulating network interface MAC addresses.
```

> - View current and permanent MAC address of interface:

Reviewer: ```suggestion
- View the current and permanent MAC addresses of a interface:
```
In case it actually displays 2 (distinct) adresses and not 1.

`macchanger --show {{interface}}`

> - Set interface to random MAC:

Reviewer: ```suggestion
- Set interface to a random MAC:
```

`macchanger --random {{interface}}`

> - Set interface to specific MAC:

Reviewer: ```suggestion
- Set interface to a specific MAC:
```

`macchanger --mac {{XX:XX:XX:XX:XX:XX}} {{interface}}`

> - Reset interface to permanent hardware MAC:

Reviewer: ```suggestion
- Reset interface to a permanent hardware MAC:
```
